installation/before-install.md

```diff
@@ -55,5 +55,22 @@ Use `sysctl -p` command or reboot your node for the changes to take effect.
 
 These kernel options were originally taken from the presentation [How Netflix Tunes EC2 Instances for Performance](https://www.slideshare.net/brendangregg/how-netflix-tunes-ec2-instances-for-performance) by [Brendan Gregg](http://www.brendangregg.com/), Senior Performance Architect at AWS re:Invent 2017.
 
-If this article is incorrect or outdated, or omits critical information, please [let us know](https://github.com/fluent/fluentd-docs-gitbook/issues?state=open). [Fluentd](http://www.fluentd.org/) is an open-source project under [Cloud Native Computing Foundation \(CNCF\)](https://cncf.io/). All components are available under the Apache 2 License.
+## Use sticky bit symlink/hardlink protection
+
+**NOTE:** CentOS 7 or later, Ubuntu 18.04 (bionic) or later, and Debian GNU/Linux 10 (buster) or later are supported these parameters.
+
+Fluentd sometimes uses predictable paths for dumping, writing files, and so on.
+For symlink attack protection, check the following parameters are set up as `1`:
```

Comment:
Should be described which file have to be modified (`/etc/sysctl.conf`).
Although it can be predictable from the previous section, should be clarified as same as `sysctl -p`.

Reply:
Ah, we should. Thanks for the pointing out.

Reply:
Fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/318/commits/318417e0d71158e3514699b7f28339df36343dad.